lib/import-from-tiled.js

```diff
@@ -0,0 +1,135 @@
+"use strict";
+
+/** <p>Import an orthogonal tilemap from
+ * <a href="http://www.mapeditor.org/" target="_blank">Tiled: Map Editor</a> and draw on the screen.</p>
+ * <h5>To Do</h5>
+ * <ul>
+ * <li><b>Image Layer</b></li>
+ * <li><b>Ellipse and Polygon Objects</b></li>
+ * </ul>
+ * @module Import From Tiled
+ */
+
+/** @function ImportTilemap
+ * @param {Object} file JSON file exported from Tiled. This should be required in a scene enter
+ * script and passed to the function.
+ * @param {EntityPool} entities EntityPool from game.entities
+ */
+function ImportTilemap(file, entities) { // eslint-disable-line no-unused-vars
+
+	var tile, image, image_index, tile_pos, cols, tileset, tileset_index = 0;
+	var layer, object, entity, object_properties = {};
+
+	for (var i = 0; i < file.layers.length; i++) {
+		layer = file.layers[i];
+
+		// Render Tile Layers
+		if (layer.type == "tilelayer") {
+			for (j = 0; j < layer.data.length; j++) {
```

Comment:
The body of this `for` loop should be extracted to a function. Something like `processTile(layer.data[j])`.